coalib/settings/ConfigurationGathering.py

```diff
@@ -162,14 +162,17 @@ def save_sections(sections):
     conf_writer.close()
 
 
-def warn_nonexistent_targets(targets, sections, log_printer=None):
+def warn_nonexistent_targets(targets, sections,
+                             only_global_bears, log_printer=None):
```

Comment:
never add parameters in the middle of the list; that breaks the API.  Add them to the end.

Reply:
@jayvdb but positional parameter should be before named parameters. Am I missing something over here?

Reply:
Maybe make it a keyword parameter and shift it to the end?
However, I think we have done deprecation efforts way too long. We are still having `0.x` versions, and as a maintainer I would claim our right to break API (as long as we properly mention it in the release notes) especially because we are officially still "alpha" ^^ Those internal things are likely to be not used outside, and people can still use the older version if they are too lazy to update. If we can pin package versions, others can do the same ;)
But @jayvdb that's a different discussion I'm kicking off here, your decision :)

Reply:
I will do it but can you explain me in what context are you all talking about? why does this cause API breakage?